includes/admin/class-wcmp-api.php

```diff
@@ -90,7 +85,7 @@ public function add_shipments(array $shipments, string $type = "standard"): arra
             "user-agent"    => $this->userAgent,
         ];
 
-        $request_url = $this->apiUrl . $endpoint;
+        $request_url = MyParcelRequest::REQUEST_URL . '/' . $endpoint;
```

Comment:
Als je hier toch bent mag je van mij stiekem de namen van variabelen in camelcase doen